src/lib/components/tabs/QTabs.svelte

```diff
@@ -24,111 +20,124 @@
     ...props
   }: QTabsProps = $props();
 
-  const cssVars = {
-    indicatorPosition: "--indicator-position",
-    indicatorSize: "--indicator-size",
-  };
-
   let qTabs: HTMLElement;
+  let tabList: HTMLElement[];
 
-  const ctx = new QContext<QTabStore>("q-tabs", {
-    value,
-    variant,
-    previousEl: null,
-    activeEl: null,
-    utils: { size: 0, position: 0 },
-  });
+  const valueContext = new QContext<string | undefined | null>("QTabsValue", value);
+  const requestContext = new QContext<string | null>("QTabsRequest", null);
+
+  // Set the variant context
+  setContext("QTabsVariant", variant);
```

Comment:
Why no `QContext`?

Reply:
Because "variant" doesn't need to be modified, it's just passing a readonly values between components: no need to create a `$state` value for that